### PR TITLE
Update test script to fail on error and remove obsolete .gitingore entry

### PR DESCRIPTION
**Pull request by lalexgap**

### CODE REVIEW

File: scripts/test_truffle.js [hunk 1]
```diff
@@ -22,9 +22,17 @@ try {
     }
   });
 
-  exec('truffle.cmd test --network ganache', (err, stdout, stderr) => {
+  exec('truffle test --network ganache', (err, stdout, stderr) => {
```
Comment (andrewgordstewart) [inline scripts/test_truffle.js]: Should this be using the project's specified version of truffle? ie. `node_modules/.bin/truffle`?

File: scripts/test_truffle.js [hunk 2]
```diff
@@ -22,9 +22,17 @@ try {
...
     // Errors seem to be piped to stdout so we just output that always
```
Comment (andrewgordstewart) [inline scripts/test_truffle.js]: Comment is no longer true.

Comment (lalexgap) [reply scripts/test_truffle.js]: Oops

File: scripts/test_truffle.js (hunk 1 above)
Comment (lalexgap) [reply scripts/test_truffle.js]: Yup, that's a good point.

**Pull request merged**